README.md

```diff
@@ -1,134 +1,32 @@
 <img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
 
-# Google Cloud Platform Node.js Samples [![Slack][slack_badge]][slack_link] [![Coverage][cov_badge]][cov_link]
+# Google Cloud Platform Node.js Samples 
 
 Node.js samples for [Google Cloud Platform products][cloud].
 
 See [cloud.google.com/nodejs][cloud_nodejs] to get up and running with Node.js
 on Google Cloud Platform.
 
-[slack_badge]: https://img.shields.io/badge/slack-nodejs%20on%20gcp-E01563.svg
-[slack_link]: https://gcp-slack.appspot.com/
-[cov_badge]: https://img.shields.io/codecov/c/github/GoogleCloudPlatform/nodejs-docs-samples/master.svg?style=flat
-[cov_link]: https://codecov.io/github/GoogleCloudPlatform/nodejs-docs-samples?branch=master
 [cloud]: https://cloud.google.com/
 [cloud_nodejs]: https://cloud.google.com/nodejs/
 
-## Table of Contents
```

Comment:
Do we want to drop the TOC?  At first I thought this PR was just yanking badges :)

Reply:
This PR is deleting lots of stuff. The TOC doesn't make sense anymore since most samples moved to their client library. I feel that the folders in the code are a better (and always up to date) TOC.